CHANGELOG.md

```diff
@@ -0,0 +1,317 @@
+ * Remove RHEL6 support in katello-change-hostname ([#20463](http://projects.theforeman.org/issues/20463), [45417cf6](http://github.com/katello/katello-packaging/commit/45417cf61d90a5c862f8fb9b1a4ede81c543b297))
+ * Error on registering a host ([#19970](http://projects.theforeman.org/issues/19970), [46bee19e](http://github.com/katello/katello/commit/46bee19ed7fe7c445ecf45c196825143e0bbf4b9))
+ * katello-change-hostname should ask for credentials if not provided ([#20805](http://projects.theforeman.org/issues/20805), [a29de727](http://github.com/katello/katello-packaging/commit/a29de727340560128e086e57113202f21accfc2a))
+ * Hypervisors show up as Hosts with no Puppet reports ([#20727](http://projects.theforeman.org/issues/20727), [70c8be75](http://github.com/katello/katello/commit/70c8be756dda2b3c92fd32b6cee81136af93eb5a))
+ * Cancel button not working on create activation key page ([#20719](http://projects.theforeman.org/issues/20719), [3fba6188](http://github.com/katello/katello/commit/3fba61885e15da7536acd1b382fdeb590bc46aa6))
+ * Content Hosts Traces page doesnt work ([#20703](http://projects.theforeman.org/issues/20703), [321c0071](http://github.com/katello/katello/commit/321c0071dcc05e0116b67c3acb46fc2a1b0a27b9))
+ * API call of GET /katello/api/host_collections says organization_id optional, but doesn't accept without organization id ([#20574](http://projects.theforeman.org/issues/20574), [6f77b2e3](http://github.com/katello/katello/commit/6f77b2e34a88eff02043298ccd062b7ee94b828e))
+ * name field not clickable after opening resource switcher ([#21035](http://projects.theforeman.org/issues/21035), [3e8b0796](http://github.com/katello/bastion/commit/3e8b07964f3066d77575e1821fbdac83180cda59))
+ * Update installation media paths in k-change-hostname ([#20987](http://projects.theforeman.org/issues/20987), [f1418fc0](http://github.com/katello/katello-packaging/commit/f1418fc08a9f2614b0f136bde6469328f0b6bf34))
+ * katello-change-hostname should print command output when it errors out. ([#20984](http://projects.theforeman.org/issues/20984), [a727125b](http://github.com/katello/katello-packaging/commit/a727125bb492abdf14c9eb10c39e8d67d9864cf9))
+ * k-change-hostname can mix up internal capsule names ([#20983](http://projects.theforeman.org/issues/20983), [b1a9d445](http://github.com/katello/katello-packaging/commit/b1a9d44581e7e93131c8cadfe04cb0b2d32b2346))
+ * clean_installed_packages script logs a count(*), causing slow performance ([#20946](http://projects.theforeman.org/issues/20946), [cc0b15fb](http://github.com/katello/katello/commit/cc0b15fb330178aca8317873eaaf2ef1b280d1d7))
+ * k-change-hostname will check for exit code on a skipped command on a foreman-proxy ([#20944](http://projects.theforeman.org/issues/20944), [d32d5854](http://github.com/katello/katello-packaging/commit/d32d5854c167efa0bf2dc02fa5ceceb07f2eff71))
+# 3.5.0 Schwarzbier (2017-09-21)
```

Comment:
Duplicated here -- seems like a bug in tool_belt

Reply:
Ah! No bug. I just realized that it amends the file, not overwrites it, so you have to remove the changelog when you regenerate it.